tools/linter_lib/custom_check.py

```diff
@@ -25,6 +25,7 @@
     "zerver/lib/send_email.py",
     "zerver/tests/test_new_users.py",
     "zerver/tests/test_email_mirror.py",
+    "zerver/tests/test_missed_message_emails.py",
```

Comment:
I would prefer a name like `test_message_notification_emails.py` -- the "missed message" framing is something from the early days that we've been trying to move away from. I picked that name to somewhat parallel `ScheduledMessageNotificationEmail`.